package.json

```diff
@@ -24,13 +24,14 @@
   "activationEvents": [
     "workspaceContains:jest.config.js",
     "workspaceContains:jest.json",
-    "workspaceContains:node_modules/.bin/jest",
-    "workspaceContains:node_modules/react-scripts/node_modules/.bin/jest",
+    "workspaceContains:node_modules/.bin/jest?(.cmd)",
+    "workspaceContains:node_modules/react-scripts/node_modules/.bin/jest?(.cmd)",
```

Comment:
> Unfortunately I didn't find any specification(...)

Specification on what exactly?

Reply:
Specifications about which files will be in the .bin folder. Currently if you install packages on Windows, it generates both, the executables without file extension (for example `jest`) and the executables with extension (`jest.cmd`). Since we don't have to, I wouldn't introduce any operating system specifics to our `package.json` (and stick to the old `workspaceContains` values).

Reply:
I'm reverting this :)